assets/constants.ts

```diff
@@ -29,6 +29,24 @@ export default {
   // fragment in the controller is invalid.
   "DEFAULT_CLASSROOM_URL_FRAGMENT": "math",
 
+  // Add classroom name to SEARCH_DROPDOWN_CLASSROOMS in this file
+  // to add that classroom to learner group syllabus filter whenever a new
+  // classroom is added.
+  "CLASSROOM_PAGES_DATA": [{
```

Comment:
Having this here might be a problem, see top-level comment. Also I think we need it beyond just for learner groups?

Reply:
Restored the classroom config property for now.